### ISSUE TITLE
Wanderer: WordPress.com theme demo screenshot features inaccessible post subtitle text option

### ISSUE THREAD

**Issue by tbradsha** [Automattic/themes#3054]
## Steps to replicate
Go to the WordPress.com theme page for Wanderer: https://wordpress.com/theme/wanderer
* The theme screenshot looks like this:
 ![image](https://user-images.githubusercontent.com/32492176/104941298-1fd20980-5970-11eb-9c47-5caef7874d3b.png)

* However, the actual theme looks like this:
 ![image](https://user-images.githubusercontent.com/32492176/104941940-01b8d900-5971-11eb-827c-d0f34ebe2615.png)

Note the lack of subtitles. The non-WordPress.com theme features those subtitles:

https://demos.press75.com/wanderer/

This is not controlled by excerpts, but instead a `meta_key` called `journal_sub_title` or `wanderer_sub_title` per the non-WordPress.com theme sample content found [here](https://demos.press75.com/themes/wp-content/themes/wanderer/sample-data.xml).

* User report: #27049693-hc

**Comment by scruffian** [comment]
Moved to Trac

**Comment by tbradsha** [comment]
@scruffian, for future reference should I have opened this directly in Trac?

**Comment by scruffian** [comment]
You don't have access to the right Trac, so its' best to log it in GitHub and then let me copy it over. Thank you.

**Comment by nickpelton** [comment]
This is incorrect. The Wanderer theme on our press75.com site is not the same codebase as the WordPress.com codebase. On wp.com we use the excerpt for the subtitle while on our press75.com version we use a meta field.  @tbradsha Please reference the .com codebase for future issue reports. Thanks!

**Comment by tbradsha** [comment]
I'm not a dev, so don't have the ability to check the codebase, but it does appear to be resolved. We did try excerpts on the site in question (that was our first guess, at which point I dug for the self-hosted docs), as well as on several test sites, but it didn't work at the time.

For example, the screenshot above shows no subtitle under the "France: A History Lesson" post (last edited 4 years ago to add an excerpt). It now shows properly:

![image](https://user-images.githubusercontent.com/32492176/105768228-516f4580-5f19-11eb-995a-dc16329da827.png)

Reopening so that someone can add that excerpt = subtitle to the docs, as well as to update the screenshot to look like the demo site.

**Comment by nickpelton** [comment]
There was a bug, but it was related to not showing the except properly, so it was still helpful. Thanks!
